treeherder/model/models.py

```diff
@@ -1074,7 +1075,8 @@ class FailureLine(models.Model):
     status = models.CharField(max_length=7, choices=STATUS_CHOICES)
     expected = models.CharField(max_length=7, choices=STATUS_CHOICES, blank=True, null=True)
     message = models.TextField(blank=True, null=True)
-    signature = models.TextField(blank=True, null=True)  # Prefix index length 50
+    # Prefix index: signature(50)
+    signature = models.TextField(blank=True, null=True, db_index=True)
```

Comment:
This looks a little funny to me since (last I checked) setting an index on a TextField normally isn't allowed. I would personally just take out the db_index, and maybe put in a more detailed comment saying that we put in a prefix index manually.